eng/Signing.props

```diff
@@ -14,6 +13,9 @@
     <FileSignInfo Include="mscordaccore.dll" CertificateName="None" />
     <FileSignInfo Include="mscordbi.dll" CertificateName="None" />
 
+    <!-- On MacOS, we need to sign a number of our executables with the Mac developer cert with hardening enabled. -->
+    <FileSignInfo Condition="'$(TargetsOSX)' == 'true'" Include="dotnet;apphost;corerun;createdump" CertificateName="MacDeveloperHarden" />
```

Comment:
Will these files have the appropriate entitlements before signing?

Reply:
Yes, we entitle and ad-hoc sign them during the initial CMake build. As long as this preserves entitlements (which as far as I can tell from docs does), we should be good.

Reply:
Yes, it will.